fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/event/lifecycle/v1/ServerEntityEvents.java

```diff
@@ -84,4 +98,9 @@ public interface Load {
 	public interface Unload {
 		void onUnload(Entity entity, ServerWorld world);
 	}
+
+	@FunctionalInterface
+	public interface EquipmentChange {
+		void onChange(LivingEntity livingEntity, EquipmentSlot equipmentSlot, ItemStack previousStack, ItemStack nextStack);
```

Comment:
```suggestion
		void onChange(LivingEntity livingEntity, EquipmentSlot equipmentSlot, ItemStack previousStack, ItemStack currentStack);
```

Could also use a javadoc comment maybe to explain that this method is called once per slot etc.